### ISSUE TITLE
[feat] devOps friendly output format (JSON, YAML)

### ISSUE THREAD

**Issue by vicary** [serverless/components#945]
# Description
It should be a common use case to integrate Serverless Component into CI/CD pipelines, where outputs are usually parsed with scripts to further trigger other pipelines.

It would be really helpful if `serverless deploy` and `serverless info` allows an option for output formats, such as `serverless deploy --format json`.

This option should implies skipping the deploying TUI progress bar, which also solves #834.

**Comment by KimPinot** [comment]
Hello, I am writing PR based on your Issue.
What information should be printed out to be of great help?

**Comment by vicary** [comment]
Thanks for making an effort. So in theory devOps pipelines should be programmatically friendly to parse, in general

1. Remove CLI progress bar

**Comment by vicary** [comment]
Thanks for making an effort. So in theory devOps pipelines should be programmatically friendly to parse, in general it should

1. Remove CLI progress bar
2. Show deployment outputs in JSON format, I am expecting the same data that can be referenced via the syntax `${output:...}` in other components.

**Comment by KimPinot** [comment]
Aha. So you want information like 'serverless info --debug' to be displayed in JSON or YAML format after the distribution pipeline is terminated?

**Comment by vicary** [comment]
Yes. Such that I can easily plug my own scripts to grep necessary information into our own CI/CD pipelines. For example I can send the ARN downstream to another repo that updates our k8s proxy.

**Comment by KimPinot** [comment]
Thanks for your help!
Your answer is more help to this PR!

**Comment by vicary** [comment]
Glad to be of help!
